app/models/SdilReturn.scala

```diff
@@ -17,6 +17,7 @@
 package models
 
 import cats.implicits._
+import config.FrontendAppConfig
```

Comment:
Should this import have been added?

Reply:
yes, but the changes were not kept, so should be removed.